src/components/player/PlayerHeroWinRate.vue

```diff
@@ -105,7 +103,7 @@ export default defineComponent({
       { text: "vs. Random", value: ERaceEnum.RANDOM },
     ];
 
-    const headersWithoutImageAndName = headers.slice(2);
+    const headersWithoutImageAndName = headers.slice(2) as { text: string; value: ERaceEnum }[];
```

Comment:
What is the benefit of this `as`?

Reply:
TS thinks it's `{ text: string; value: ERaceEnum } | { text: string; value: string }` without this, because that's effectively what `headers` has. Then the template above at `<td v-on="on" v-html="item[header.value]"></td>` would complain cause it doesn't know how to use `string` as an index into an array (strings can index into an array).

```
Element implicitly has an 'any' type because expression of type 'string | ERaceEnum' can't be used to index type 'PlayerHeroWinRateForStatisticsTab'.
  No index signature with a parameter of type 'string' was found on type 'PlayerHeroWinRateForStatisticsTab'.ts-plugin(7053)
```